commit 941af2693705635577a04e883ade205d45420ccb (refs/changes/70/4570/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:26:53+00:00 (1 year, 2 months ago)
    
    MANTA-3753 muppet should leverage autometadata instead of checking nictags (bump version)

diff --git a/package.json b/package.json
index da15204..9a5cbe0 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "muppet",
     "description": "Joyent's Load Balancer",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
